virtualpdu/main.py

```diff
@@ -26,6 +27,8 @@
 
 MISSING_CONFIG_MESSAGE = 'Missing configuration file as first parameter.\n'
 
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
PEP8 hates that. Will have to think of a way to still meet this requirement while keeping PEP8 happy.